pom.xml

```diff
@@ -446,6 +447,7 @@
               <exclude>.classpath</exclude>
               <exclude>.project</exclude>
               <exclude>**/dependency-reduced-pom.xml</exclude>
+              <exclude>**/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister</exclude>
```

Comment:
@lresende even though [java.util.ServiceLoader](http://docs.oracle.com/javase/7/docs/api/java/util/ServiceLoader.html) ignores `#`-comment lines I did not see any examples of it being used in DataSourceRegister files of other projects &mdash; i.e. Spark does not include license comments in its 3 DataSourceRegister service provider configuration files. What's wrong with excluding it from the RAT license checks?